### PR TITLE
Unit Tests: Coverage Increase

### PR DESCRIPTION
**Pull request by georgiyazovaliiski**
Signed-off-by: Georgi Yazovaliyski <georgi.yazovaliiski@gmail.com>

### CODE REVIEW

**Review by joan41868 — APPROVED** [review]
LGTM

File: hedera-node/src/test/java/com/hedera/services/store/contracts/DWUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.hedera.services.store.contracts;
+
+import org.apache.tuweni.units.bigints.UInt256;
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/store/contracts/DWUtilTest.java:3]: nit: license

File: hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExceptionalHaltReasonTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*-
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExceptionalHaltReasonTest.java:1]: license should be below the package

File: hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
+/*-
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java:1]: license should be below the package

File: hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import java.util.Optional;
+
+import static org.hyperledger.besu.evm.frame.ExceptionalHaltReason.INSUFFICIENT_STACK_ITEMS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+public class HederaExtCodeSizeOperationTest {
+    final private String ethAddress = "0xc257274276a4e539741ca11b590b9447b26a8051";
+    final private String ethRecipientAddress = "0xc257274276a4e539741ca11b590b9447b26a8052";
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java:46]: This is not used

File: hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaSelfDestructOperationTest.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
+/*-
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaSelfDestructOperationTest.java:1]: license should be below the package

File: hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    void setUp() {
+        subject = new HederaExtCodeSizeOperation(gasCalculator);
+    }
+
+    @Test
+    void executeWorldUpdaterResolvesToNull() {
+        given(mf.getStackItem(0)).willReturn(ethAddressInstance);
+        given(mf.getWorldUpdater()).willReturn(worldUpdater);
+        given(gasCalculator.getExtCodeSizeOperationGasCost()).willReturn(Gas.of(10L));
+        given(gasCalculator.getWarmStorageReadCost()).willReturn(Gas.of(2L));
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaExtCodeSizeOperationTest.java:74]: You can extract those two in `setup`

File: hedera-node/src/test/java/com/hedera/services/txns/contract/process/HederaBlockValuesTest.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
+/*-
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/process/HederaBlockValuesTest.java:1]: license should be below the package

File: hedera-node/src/test/java/com/hedera/services/utils/MerkleEntityIdUtilsTest.java [hunk 8]
```diff
@@ -180,6 +181,26 @@ void asContractWorks() {
 		assertEquals(expected, cid);
 	}
 
+	@Test
+	void asAccountWorks() {
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/utils/MerkleEntityIdUtilsTest.java:185]: `asAccountFromContractWorks`

File: hedera-node/src/test/java/com/hedera/services/txns/contract/process/HederaTracerTest.java [hunk 9]
```diff
@@ -0,0 +1,43 @@
+package com.hedera.services.txns.contract.process;
+
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/process/HederaTracerTest.java:2]: nit: license

**Review by CordonaCodeCraft — APPROVED** [review]
OLGTM

**Review by stoqnkpL — APPROVED** [review]

**Pull request merged**